### DISCUSSION TITLE
Excalidraw live collaboration

### DISCUSSION THREAD

**Discussion by Sinkler521** [excalidraw/excalidraw#8487] [General]
Greetings, and thank you so much for the great job you've done here. The editor gives really pleasant user experience and simple to use.

One problem I've faced: seems like it has no _live collaboration_ built-in func, despite the website's editor has. I've read an article in Excalidraw blog and understand how many difficulties with connection and correct work can appear while developing and testing this feature, so I really ask you for help.

**If you can share such _live collaboration_ feature source or give me some instructions to help me avoid developing that, I would be grateful.**

I also found this link in one topic related to same problem: https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/collab/CollabWrapper.tsx#L387, but seems like in actual master branch there are no files like that, despite the library has some components that imply that we will implement (such as LiveCollaborationTrigger component or collaborators array as a part of appState).
If you have a solution, please let me know. Thanks

**Comment by xvalentine** [comment]
Hi in order for you to go live you need to run:

https://github.com/excalidraw/excalidraw-room

Yarn start:dev 

And it should automatically function.

How I found this: I went to excalidraw docs and searched collab.

You can use pm2 to keep the server running and you shouldn't lose the data (not 100% sure for how long though) 

Hope this helps
